File: CS/ExtendModel.Module/ModelExtender.cs
using DevExpress.Data;
using System.ComponentModel;

namespace ExtendModel.Module {
    public interface IModelListViewExtender {
        bool IsGroupFooterVisible { get;set;}
    }
    public interface IModelColumnExtender {
        [DefaultValue(SummaryItemType.None)]
        SummaryItemType GroupFooterSummaryType { get;set;}
    }
}

File: VB/ExtendModel.Module/ModelExtender.vb
Imports DevExpress.Data
Imports System.ComponentModel

Namespace ExtendModel.Module
	Public Interface IModelListViewExtender
		Property IsGroupFooterVisible() As Boolean
	End Interface
	Public Interface IModelColumnExtender
		<DefaultValue(SummaryItemType.None)>
		Property GroupFooterSummaryType() As SummaryItemType
	End Interface
End Namespace
